lib/entities/quote/RelayQuote.ts

```diff
@@ -52,6 +52,31 @@ export class RelayQuote implements IQuote {
   public readonly createdAt: string;
   public derived: RelayQuoteDerived;
   public routingType: RoutingType.RELAY = RoutingType.RELAY;
+  // Add 1bps price improvmement to favor Relay over classic
+  public static amountOutImprovementExactIn = BigNumber.from(10001);
+  public static amountInImprovementExactOut = BigNumber.from(9999);
```

Comment:
One problem of doing this approach is that now both Dutch and Relay quotes are 1bp better than classic :/ so doesn't fit well into the current way we compare quotes. Perhaps we should write custom special case logic for comparing quotes instead of relying on this arbitrary price improvement via bps